Peer.UnitTests/Github/GithubPeerConfigDtoTests.cs

```diff
@@ -3,7 +3,7 @@
 using Peer.GitHub;
 using Xunit;
 
-namespace Peer.UnitTests.GitHub
+namespace Peer.UnitTests.Github
```

Comment:
I think GitHub is right isn't it?

Reply:
You're right I think the namespace just got corrected to the folder name which is wrong, I'll fix both